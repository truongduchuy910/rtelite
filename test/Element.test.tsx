import renderer from "react-test-renderer";
import ElementRender from "../dist/render/Element";

test("ElementRender", () => {
  const component = renderer.create(
    <ElementRender element={{ tag: "section" }}>test</ElementRender>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
