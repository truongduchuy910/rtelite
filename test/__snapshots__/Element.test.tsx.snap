// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ElementRender 1`] = `
<section>
  test
</section>
`;
